app/abilities/project.js

```diff
@@ -12,6 +12,15 @@ import projectMember from 'code-corps-ember/macros/project-member';
 export default Ability.extend({
   currentUser: service(),
 
+  /**
+   * An `ember-can` ability.
+   *
+   * Indicates if the current user can manage a project.
+   * Returns true if the user is the owner of the project.
+   * @type {Boolean}
+   */
+  canAdminister: or('{userIsAdmin,userIsOwner}'),
```

Comment:
Until this PR, all of the bigger actions on a project were accessible by owners only. The only thing admins could do was approve pending members.